packages/renderer/src/lib/deployments/DeploymentColumnConditions.svelte

```diff
@@ -9,26 +18,64 @@ export let object: DeploymentUI;
 
 // Determine both the icon and color based on the deployment condition
 function getConditionAttributes(condition: DeploymentCondition) {
-  let name = condition.type;
-  switch (condition.type) {
-    case 'Available':
-      // faCheckCircle: Indicates a successful state, typically used to denote availability and operational readiness
-      return { color: 'text-green-600', icon: faCheckCircle, name };
-    case 'Progressing':
-      // faSync: Often used to represent ongoing processes or operations, fitting for a "Progressing" state
-      // If reason has NewReplicaSetAvailable then it's progressed
-      if (condition.reason === 'NewReplicaSetAvailable') {
-        name = 'Progressed';
-      }
+  const defaults = {
+    name: condition.type,
+    color: 'text-gray-500',
+    icon: faQuestionCircle,
+  };
 
-      return { color: 'text-sky-400', icon: faSync, name };
-    case 'ReplicaFailure':
-      // faExclamationTriangle: Alerts and warnings
-      return { color: 'text-amber-600', icon: faExclamationTriangle, name };
-    default:
-      // faQuestionCircle: Uncertain / unknown
-      return { color: 'text-gray-900', icon: faQuestionCircle, name };
-  }
+  // Condition map for easier lookup
```

Comment:
need to check how these colors would behave in light mode.

And I guess that we should no longer use tailwind color but rather use CSS variable property

Reply:
You are right, I should be using https://github.com/containers/podman-desktop/blob/b6d146ab712018c5e834f4f40d3ff2a974b7aa94/tailwind.config.cjs#L49 I'll update the PR.

Reply:
no, you should use the color-registry.ts now (tailwind should only contains the palette colors)